pkg/liqoctl/uninstall/handler.go

```diff
@@ -66,7 +66,10 @@ func (o *Options) Run(ctx context.Context) error {
 
 	s := o.Printer.StartSpinner("Running pre-uninstall checks")
 	if err := utils.PreUninstall(ctx, o.CRClient); err != nil {
-		s.Fail("Pre-uninstall checks failed: ", output.PrettyErr(err))
+		errMsg := fmt.Sprintf("Pre-uninstall checks failed: %s\n"+
```

Comment:
Are we checking here also for offloaded namespaces? if so, I would add the command to unoffload namespaces too

Reply:
Fixed